prac_05/state_names.py

```diff
@@ -0,0 +1,20 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+print(CODE_TO_NAME)
+for item in CODE_TO_NAME:
+    print(f"{item:<3} is {CODE_TO_NAME[item]}")
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    try:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+        state_code = input("Enter short state: ").upper()
```

Comment:
If you prompt the user outside of the try statement, you might not have to do it twice 😄